WordPress/Tests/NamingConventions/PrefixAllGlobalsUnitTest.1.inc

```diff
@@ -674,3 +674,6 @@ class WP_Atom_Server {
 }
 
 // phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[]
+
+// phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[] ^%&
+function ^%&_do_something() {}
```

Comment:
1. This is not a valid test for the minimum char limit as this is an invalid prefix no matter what.
2. The test is placed incorrectly, it needs to be **above** the reset of the prefixes on line 676. Custom properties always need to be reset at the end of each test file.
3.. Always have a new line at the end of every file.